.github/workflows/build_test_deploy.yml

```diff
@@ -469,6 +469,43 @@ jobs:
       - run: xvfb-run node run-tests.js test/integration/with-electron/test/index.test.js
         if: ${{needs.build.outputs.docsChange != 'docs only change'}}
 
+  testPartytown:
```

Comment:
Instead of having this as a separate job we can use the new test setup that allows installing extra test dependencies, here's an example test where this is being done https://github.com/vercel/next.js/blob/2c7f8b3d7b098d5f0847662c1b1ee1a10f1efbc1/test/production/next/jest/index.test.ts#L105

Reply:
Oh sweet, I wasn't aware of this 😍

Removed the separate job and [used this setup](https://github.com/vercel/next.js/pull/34244/files#diff-65fbfdfb8a4bbccb153d9b47ed0697d124c167a94d729d8b758cc943fdef6def) instead.